src/lib.rs

```diff
@@ -64,9 +65,10 @@ fn resp_to_rows(obj: &str, resp: &JsonValue, quals: &[Qual]) -> Vec<Row> {
 }
 
 #[wrappers_fdw(
-    version = "0.1.5",
+    version = "0.2.0",
     author = "Jay Kothari",
-    website = "https://tembo.io"
+    website = "https://tembo.io",
+    error_type = "PrometheusFdwError"
```

Comment:
Can you have a trailing comma on the last item in the list? Helps prevent weird diffs like this one where the website line had to have a comma added.

If not, I wouldn't hold up this PR to add it, mind you.

Reply:
good call